### PR TITLE
Include LB prerequisites directly

### PR DESCRIPTION
**Pull request by asteflova**
Currently, the Load Balancing guide links to sections in the Installing Proxy guide. That seems odd because with modular documentation, we can include these sections directly. In this PR, I propose these changes:

* Create an assembly file for preparing a Smart Proxy for LB
* Fill in the assembly with includes pointing to the relevant modules for installing a proxy

I'm including the modules with the conditionals that are also used in the Installing Proxy's master.adoc to make it work for other builds if needed.

* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [ ] Foreman 3.10/Katello 4.12
* [ ] Foreman 3.9/Katello 4.11 (planned Satellite 6.15; orcharhino 6.8)
* [ ] Foreman 3.8/Katello 4.10
* [ ] Foreman 3.7/Katello 4.9 (Satellite 6.14)
* [ ] Foreman 3.6/Katello 4.8
* [ ] Foreman 3.5/Katello 4.7 (Satellite 6.13; orcharhino 6.6/6.7)
* [ ] Foreman 3.4/Katello 4.6 (EL8 only)
* [ ] Foreman 3.3/Katello 4.5 on EL7 & EL8 (Satellite 6.12 on EL8 only; orcharhino 6.4/6.5 on EL8 only)
* We do not accept PRs for Foreman older than 3.3.

### CODE REVIEW

File: guides/common/modules/con_prerequisites-for-configuring-smart-proxy-server-for-load-balancing.adoc [hunk 1]
```diff
@@ -1,16 +1,4 @@
 [id="Prerequisites_for_Configuring_{smart-proxy-context}_Servers_for_Load_Balancing_{context}"]
 = Prerequisites for configuring {SmartProxyServers} for load balancing
 
-ifdef::orcharhino[]
 You can find a list of requirements for {SmartProxyServer} in xref:sources/installation_and_maintenance/installing_orcharhino_proxy.adoc[_{InstallingSmartProxyDocTitle}_].
```
Comment (asteflova) [inline guides/common/modules/con_prerequisites-for-configuring-smart-proxy-server-for-load-balancing.adoc:4]: @maximiliankolb I noticed you're doing things differently for this part of the docs for orcharhino so I left this file in just for orcharhino. (I moved the conditionals from here to LB/master.adoc).

File: guides/common/modules/ref_preparing-smartproxyservers-for-load-balancing-next-steps.adoc [hunk 2]
```diff
@@ -0,0 +1,4 @@
+[id="preparing-{smart-proxy-context}-servers-for-load-balancing-next-steps_{context}"]
+= Next steps
```
Comment (maximiliankolb) [inline guides/common/modules/ref_preparing-smartproxyservers-for-load-balancing-next-steps.adoc:2]: Any reason to not go with ".Next steps"? same above for "additional resources".

**Review by maximiliankolb — COMMENTED** [review]
some questions; rest of the diff LGTM

File: guides/common/modules/con_preparing-smartproxyservers-for-load-balancing.adoc [hunk 3]
```diff
@@ -0,0 +1,5 @@
+[id="preparing-{smart-proxy-context}-servers-for-load-balancing_{context}"]
+= Preparing {SmartProxyServers} for load balancing
+
+{Project} does not support configuring existing {SmartProxyServers} for load balancing.
+You must configure a new {SmartProxyServer} for this purpose.
```
[the remark this answers is not in the document]
Comment (asteflova) [reply guides/common/modules/con_preparing-smartproxyservers-for-load-balancing.adoc]: Let's go with create.

File: guides/common/modules/ref_preparing-smartproxyservers-for-load-balancing-next-steps.adoc (hunk 2 above)
Comment (maximiliankolb) [reply guides/common/modules/ref_preparing-smartproxyservers-for-load-balancing-next-steps.adoc:2]: The headings and therefore the navigation looks a bit off (very short and generic), but I totally get your reasoning and I'm OK with it.

Comment (asteflova) [reply guides/common/modules/ref_preparing-smartproxyservers-for-load-balancing-next-steps.adoc:2]: Thanks for voicing your opinion @maximiliankolb I get it, I agree, but I don't have a better solution that would work for what I'm trying to do. I'm happy to revisit this in the future if we come up with something better.

**Pull request merged**